### PR TITLE
SISRP-38422 - Refactors Profile and Academic Summary cards

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.berkeley.edu/browse/SISRP-38422

Includes visual updates per the design mockups.

Makes some slight changes to styling on Profile and Academic Summary to make the two cards consistent with each other.

### CODE REVIEW

**Comment by lyttam** [comment]
Will fix build failures and amend.

**Comment by lyttam** [comment]
Ready to review/merge.

File: src/assets/stylesheets/_academic_summary.scss [hunk 1]
```diff
@@ -1,7 +1,10 @@
-.cc-academic-summary {
+div.cc-academic-summary {
   .cc-academic-summary-border-wrapper {
```
Comment (redconfetti) [inline src/assets/stylesheets/_academic_summary.scss]: @lyttam - Is this `div` specifier necessary? What about the `h3`, `table`, and `options` below. Something that we've tried to maintain in the past is that we try to use the [minimum amount of specificity](https://css-tricks.com/strategies-keeping-css-specificity-low/), starting with just applying classes to change the styling of an element... and using two consecutive classes such as `.cc-academic-summary .cc-academic-summary-header` to be more specific, avoiding element and ID specifiers as much as we can.

It also encourages the [OOCSS pattern](https://www.smashingmagazine.com/2011/12/an-introduction-to-object-oriented-css-oocss/) also.

Comment (lyttam) [reply src/assets/stylesheets/_academic_summary.scss]: Actually, I did introduce it... will amend.

**Pull request merged**